### ISSUE TITLE
bug: pfe-clipboard does not have an accessible experience

### ISSUE THREAD

**Issue by kelsS** [patternfly/patternfly-elements#1873]
## Description of the issue

The component is only usable by mouse and for sighted users.

### Impacted component(s)

- pfe-clipboard

### Steps to reproduce

1. Go to a page with [pfe-clipboard](https://patternflyelements.org/elements/pfe-clipboard/demo/)
2. Turn on the screen reader
3. Attempt to copy to the clipboard and paste using screen reader only

### Expected behavior

The screen reader user should have an equivalent way to understand and use the component.

## Related issues
- #1868

**Comment by bennypowers** [comment]
Closed by #1887
